run-prover.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# USAGE examples: 
+  # CLI with env vars: MINER_ADDRESS=aleoABCD...  OPERATOR_IP_ADDRESS=a.b.c.d ./run-prover.sh
+  # CLI with prompts for vars:  ./run-prover.sh
+
+
+# if env var MINER_ADDRESS is not set, prompt for it
+if [ -z "${MINER_ADDRESS}" ]
+then
+  read -r -p "Enter your miner address: "
+  MINER_ADDRESS=$REPLY
+fi
+
+if [ "${MINER_ADDRESS}" == "" ]
+then
+  MINER_ADDRESS="aleo1d5hg2z3ma00382pngntdp68e74zv54jdxy249qhaujhks9c72yrs33ddah"
+fi
+
+# if env var OPERATOR_IP_ADDRESS is not set, prompt for it
+if [ -z "${OPERATOR_IP_ADDRESS}" ]
+then
+  read -r -p "Enter your Operator Servers IP address: "
+  OPERATOR_IP_ADDRESS=$REPLY
+fi
+
+if [ "${OPERATOR_IP_ADDRESS}" == "" ]
+then
+  echo "IP Address of Operator server is required to run a prover"
+  exit 1
+fi
+
+COMMAND="cargo run --release -- --prover ${MINER_ADDRESS} --pool ${OPERATOR_IP_ADDRESS}:4132 --trial --verbosity 2"
```

Comment:
I would not assume it is running on port 4132 so make OPERATOR_ADDRESS == IP+port.

Reply:
What do you think of this instead?  Allow for an optional ENV VAR, and if not found then set port = 4132.

Reply:
Just added this functionality to make the port changeable via env vars